release notes/v0.34.0.md

```diff
@@ -0,0 +1,59 @@
+
+```
+import exec from "k6/execution";
+import { SharedArray } from "k6/data";
+
+const data = new SharedArray("my dataset", function(){
+	// generate and return the data
+})
+
+export const options = {
+    scenarios :{
+        "use-all-the-data": {
+            executor: "shared-iterations",
+            vus: 100,
+            iterations: data.length,
+            maxDuration: "1h"
+        }
+    }
+}
+
+export default function() {
+    // this is unique even in the cloud
+    var item = data[exec.scenario.iterationInTest];
+    http.post("https://httpbin.test.k6.io/amazing-endpoint", item)
+}
+```
+
+**Docs**: [grafana/xk6-execution](https://github.com/grafana/xk6-execution)
+
+## Enhancements and UX improvements
+
+* Warn Windows users on importing dependencies or opening files as absolute paths ([#2078](https://github.com/grafana/k6/pull/2078)).
+* Pass setup data object into [handleSummary callback](https://k6.io/docs/results-visualization/end-of-test-summary/#handlesummary-callback) ([#2103](https://github.com/grafana/k6/pull/2103)). Thanks, @SamuelJohnson01997!
+
+## Breaking changes
+
+* The deprecated outputs Datadog and Kafka have been removed ([#2081](https://github.com/grafana/k6/pull/2081)).
+
+## Bugs fixed!
+
+* Use HTTP POST method instead of GET for pushing logs requests to Loki ([#2100](https://github.com/grafana/k6/pull/2100)).
+* Encode the BlacklistIP option using the CIDR notation ([#2083](https://github.com/grafana/k6/pull/2083)).
+* `ext.loadimpact` option has the same precedence as the script configuration during the consolidation process ([#2099](https://github.com/grafana/k6/pull/2099)).
+* The WebSocket connection used for tailing logs from the k6 Cloud is reestablished in the case of an unexpected error ([#2090](https://github.com/grafana/k6/pull/2090)).
+
+## Internals
+
+* A simpler and clearer API has been added as an alternative to `common.Bind`, which also gives JS modules easy access to some [useful internal objects and runtime information](https://pkg.go.dev/go.k6.io/k6@v0.34.0/js/modules#Instance) ([#2108](https://github.com/grafana/k6/pull/2108)).
+* Speeding ups TC39 tests using a pool of Babel compilers ([#1839](https://github.com/grafana/k6/pull/1839)).
+* Goja dependency has been updated adding the native support for Arrow functions, Destructuring, Default arguments and Computed properties features. For the same reason, the relative Babel's plugins supporting those features are not required anymore so they have been disabled ([#2109](https://github.com/grafana/k6/pull/2109)).
```

Comment:
This should probably be in the `Enhancements and UX improvements` category, right? After all, we now support some destructuring modes that we didn't support before, even with Babel, right @MStoykov?

Reply:
I guess we do now support https://github.com/grafana/k6/issues/824, but apart from that almost everything else is *very* minor so I wouldn't really qualify it as that much of an improvement, but we can move it

Reply:
it's certainly not an entirely internal thing, if we now support some JS aspect we didn't before, so let's move at least that part

We can also keep a mention here that we've updated some dependencies and link both to the goja update PRs and to https://github.com/grafana/k6/pull/2092

Reply:
Updated. @MStoykov WDYT? Is it clear?